server/schedule/operator_controller.go

```diff
@@ -51,6 +51,12 @@ type HeartbeatStreams interface {
 	SendMsg(region *core.RegionInfo, msg *pdpb.RegionHeartbeatResponse)
 }
 
+// WaitingOperatorStatus is used to limit the count of each kind of operators.
```

Comment:
Why do not define it in `waiting_operator.go`? I guess it is specific to waiting operators.